src/app/time-period.ts

```diff
@@ -0,0 +1,131 @@
+/**
+ * Each value of this enum represents a particular month of the year, from
+ * January to December.
+ *
+ * These values are integers: 1 is January, 2 is February, and so on.
+ */
+export enum Month {
+  January = 1,
+  February,
+  March,
+  April,
+  May,
+  June,
+  July,
+  August,
+  September,
+  October,
+  November,
+  December,
+}
+
+/**
+ * Each value of this type represents a quarter-of-the-month: 1 is the first
+ * quarter, 2 is the second quarter, and so on.
+ */
+export type Quarter = 1 | 2 | 3 | 4;
```

Comment:
I don't know if we want `quarter` to start with 1.

Reply:
I hear you, but I feel like it would be weird if `Month` and `Quarter` used different conventions :-)

Reply:
I agree with @helloworld12321 - let's go with quarters being numbered 1, 2, 3, 4 and take on the burden of any additional math so that people/elements using the API don't have to take care of that. I think it's more consistent (as @helloworld12321 said).